### DISCUSSION TITLE
API

### DISCUSSION THREAD

**Discussion by ghost** [community/community#137160] [Programming Help]
### Select Topic Area

Question

### Body

What is the best way to learn and even understand API . I've been learning python and can now write nice codes that can do what I want them to do without any problems. But when it comes to understand API I don't know where to start. I've trying for days but I can't get any idea.

**Comment by gavinwale** [comment]
This is a really good video that helped me:

https://www.youtube.com/watch?v=4vLxWqE94l4

When it comes to python, I've heard FastAPI is really good:
https://www.youtube.com/watch?v=SORiTsvnU28
https://fastapi.tiangolo.com/

**Comment by ebndev** [comment]
Hi @Avhonaho, thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our `Programming Help` category, as this topic may be more relevant there and might help someone give you a nudge in the right direction. 😁

**Comment by ghost** [comment]
Thanks

**Comment by ghost** [comment]
Great ,thank you

**Comment by xsxo** [comment]
if you mean web scraping start learn requests library
and if you mean backend servers start with any backend framwork
